templates/event.php

```diff
@@ -7,6 +7,8 @@
 
 use Wporg\TranslationEvents\Event\Event_End_Date;
 use Wporg\TranslationEvents\Event\Event_Start_Date;
+use Wporg\TranslationEvents\Stats\Event_Stats;
+use Wporg\TranslationEvents\Stats\Stats_Row;
```

Comment:
What's the reason for these? Are they added by PhpStorm?

Reply:
PhpStorm complains indeed if these aren't there. I actually thought they were required but I just tested without and it works correctly. How can it even work without it? 🤔 

Because we're using namespaces, without `use` directives, those types should be found in namespace of the current file, which is not the case.